### PR TITLE
feat: add github action integration

### PR DESCRIPTION
**Pull request by luisfelipec95**
## Description
This PR adds the configuration files to integrate github action https://github.com/eduNEXT/integration-test-in-tutor

## How to test
Checks:

- Integration / integration-test (<17.0.0) (pull_request)

- Integration / integration-test (<18.0.0) (pull_request)

## Other information

https://edunext.atlassian.net/browse/DS-963

### CODE REVIEW

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Integration
```
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply .github/workflows/integration-test.yml:1]: Or as @magajh mentioned here: https://github.com/eduNEXT/eox-tenant/pull/207/files#r1622449787 would be even better.

File: eox_hooks/tests/tutor/integration.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+echo "Install package"
+pip install -e ../eox-hooks
+
+# Install test requirements from openedx
+echo "Install test-requirements"
+make test-requirements
+
+# Running the tests using the tutor settings
+echo "Run tests"
+pytest -s --ds=lms.envs.tutor.test /openedx/eox-hooks/eox_hooks/tests/tutor
```
[the remark this answers is not in the document]
Comment (mariajgrimaldi) [reply eox_hooks/tests/tutor/integration.sh:13]: Does this work? I'm not sure in which path this is being executed.
```suggestion
pip install -e ../eox-hooks

# Install test requirements from openedx
echo "Install test-requirements"
make test-requirements

# Running the tests using the tutor settings
echo "Run tests"
pytest -s --ds=lms.envs.tutor.test /openedx/eox-hooks/eox_hooks/tests/tutor
```
```suggestion
pip install -e $(pwd)/eox-hooks

# Install test requirements from openedx
echo "Install test-requirements"
make test-requirements

# Running the tests using the tutor settings
echo "Run tests"
pytest -s --ds=lms.envs.tutor.test  $(pwd)/eox_hooks/tests/tutor
```

File: .github/workflows/integration-test.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (luisfelipec95) [reply .github/workflows/integration-test.yml]: thank you! corrected in all PR's

File: eox_hooks/tests/tutor/integration.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (luisfelipec95) [reply eox_hooks/tests/tutor/integration.sh]: with $(pwd) it didn't work for me but I left both the same

File: .github/workflows/integration-test.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Tutor Integration Tests
+on: [pull_request]
+
+jobs:
+  integration-test:
```
Comment (mariajgrimaldi) [inline .github/workflows/integration-test.yml:5]: Can we name the job so `integration-test` is not used as default? We can use the same `Tutor Integration Test` so it looks better here: https://github.com/eduNEXT/eox-hooks/actions/runs/9392586481/job/25867080058?pr=52. It'd look duplicated in the CI checks, but I don't think that would be a problem. See examples here: 
image

**Review by mariajgrimaldi — DISMISSED** [review]
Last non-blocking comment. Thank you for addressing all my concerns!

File: .github/workflows/integration-test.yml (hunk 3 above)
Comment (luisfelipec95) [reply .github/workflows/integration-test.yml:5]: Done, thanks

**Review by mariajgrimaldi — APPROVED** [review]

**Review by BryanttV — APPROVED** [review]

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**